### PR TITLE
Homework5: Andrey Kuksik

### PR DESCRIPTION
**Pull request by Kuxsoger**

### CODE REVIEW

File: src/homework5/imdb.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    try:
+        movies = []
+        with open(filename, 'r') as file:
+            for i in range(28):
+                file.readline()
+
+            for i in range(250):
+                movie = [i for i in file.readline().strip().split(' ') if i]
+                movies.append({
+                    "title": ' '.join(movie[3:-1]),
```
Comment (belovmd) [inline src/homework5/imdb.py:25]: a lot of magic numbers.

File: src/homework5/imdb.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        print("File not found")
+
+
+def write_titles(filename, data):
+    with open(filename, 'w') as file:
+        for movie in data:
+            file.write(movie["title"] + '\n')
+
+
+def create_histogram(data, key):
```
Comment (belovmd) [inline src/homework5/imdb.py:40]: please put what is kept in result dict to docstring

**Review by belovmd — COMMENTED** [review]
please address these minor issues

File: src/homework5/imdb.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                })
+        return movies
+    except IOError:
+        print("File not found")
+
+
+def write_titles(filename, data):
+    with open(filename, 'w', encoding="utf_8") as file:
+        for movie in data:
+            file.write(movie["title"] + '\n')
```
Comment (belovmd) [inline src/homework5/imdb.py:37]: redundant \\n in the end of file

File: src/homework5/imdb.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    histo = {}
+    for movie in data:
+        histo[movie[key]] = histo.get(movie[key], 0) + 1
+    return histo
+
+
+def write_histogram(filename, histo, symbol='#'):
+    keys = sorted(histo.keys())
+    width = max([len(str(key)) for key in histo]) + 1
+    with open(filename, 'w') as file:
```
Comment (belovmd) [inline src/homework5/imdb.py:51]: file is awful naming

**Review by belovmd — COMMENTED** [review]
8,5

File: src/homework5/imdb.py (hunk 4 above)
Comment (Kuxsoger) [reply src/homework5/imdb.py:51]: why? python 3 has no built-ins or keywords named “file”

**Pull request closed without merging**